### PR TITLE
Added image specs and cross reference info.

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: styleguide.md [hunk 1]
```diff
@@ -130,3 +138,10 @@ Not every single page or step needs an accompanying screenshot. When in doubt, f
 ### File format and naming:
 
 Please use .png, .gif, or .jpg files. Give the screenshots unique, meaningful names, obeying a specific format, such as general_settings_privacy. Avoid spaces, uppercase letters, and excessively long names.
+
+### Alt text:
+
+As a best practice, all screenshots should have accompanying alt text. 
+
+Use the following format when inserting an image into the docs:
+
```
Comment (Whatthefoxsays) [inline styleguide.md]: I couldn't find an actual screenshot/image with alt text (oops). Can you help me figure out the right format please and thank you?

Comment (ksigler7) [reply styleguide.md]: We include the brackets for it, we just don't currently populate it. Oops.
So here's our current text expander image:
``
and here's what it should be with alt text:
`alt text here`

**Comment by ksigler7** [comment]
other than the image example, this looks good. I'll fix the conflict really quick.

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**